src/observer.ts

```diff
@@ -32,7 +32,12 @@ export class Observer<T> implements Observable<T>
         this._event = event.trim();
     }
     
-    
+    /**
+     * 
+     * Returns a subscription to an event
```

Comment:
This does not help at all... First, the class level doc then what this function is about